1.9/administration/installing/custom/system-requirements/index.md

```diff
@@ -24,64 +24,38 @@ The cluster nodes are designated Mesos masters and agents during installation.
 
 Here are the master node hardware requirements.
 
-<table class="table">
-  <tr>
-    <th>
-      Minimum
-    </th>
-
-    <th>
-      Recommended
-    </th>
-  </tr>
-
-  <tr>
-    <td>
-      Nodes: 1<br />OS: RHEL/CentOS 7.2<br />Processor: 4 cores<br />Memory: 32 GB RAM<br />Hard disk space: 120 GB
-    </td>
-
-    <td>
-      Nodes: 3<br />OS: RHEL/CentOS 7.2<br />Processor: 4 cores<br />Memory: 32 GB RAM<br />Hard disk space: 120 GB
-    </td>
-  </tr>
-  <tr>
-     <td colspan="2">
-      <p>There are many mixed workloads on the masters, for example Mesos replicated log and ZooKeeper. Some of these require fsync()ing every so often, and this can generate a lot of very expensive random I/O. We recommend the following: <ul><li>Solid-state drive (SSD)</li><li>RAID controllers with a BBU</li><li>RAID controller cache configured in writeback mode</li></ul></p>
-      </td>
-      </tr>
-</table>
+|             | Minimum   | Recommended |
+|-------------|-----------|-------------|
+| RHEL/CentOS | 7.2, 7.3  | 7.2, 7.3    |
+| CoreOS      | 1235.9.0  | 1235.9.0    |
+| Nodes       | 1         | 6 or more   |
```

Comment:
This is under the "master node hardware requirements", so "Recommended" should be "3 or 5", not 6+ (the agent node recommendation)